CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ For more details refer to the [field mapping help page](http://help.jabref.org/e
 The new default removes the linked file from the entry instead of deleting the file from disk. [#3679](https://github.com/JabRef/jabref/issues/3679)
 
 ### Fixed
+- Fixed three commans and one and issue [#294]
```

Comment:
Would you mind describing the fix in more detail?

Reply:
Please use the format `[koppor#294](https://github.com/koppor/jabref/issues/294)`